INSTALL.md

```diff
@@ -53,14 +60,23 @@ docker exec -it [container_id] bash
 # Move to the application root dir
 cd src
 
+# Run development server
+python manage.py runserver 0.0.0.0:8080
+
+# Access to the development server
+http://0.0.0.0:8080
+
+# Access to Nginx server
+http://127.0.0.1
+
 # Run existing migrations
-python3 manage.py migrate
+python manage.py migrate
```

Comment:
Could you explain in the pull request why the python3 -> python change? IMO, `python3` is clearer.